src/main/kotlin/graphql/kickstart/tools/SchemaParser.kt

```diff
@@ -363,8 +363,8 @@ class SchemaParser internal constructor(
     private fun buildDefaultValue(value: Value<*>?): Any? {
         return when (value) {
             null -> null
-            is IntValue -> value.value
-            is FloatValue -> value.value
+            is IntValue -> value.value.toInt()
+            is FloatValue -> value.value.toDouble()
```

Comment:
This also resolves #344. 
Previously Jackson was able to convert `BigInteger` to `int` kind of accidentally I think. 
So it would work when you input the values but not when you omit them and want to use the default. 
Now that trick doesn't work anymore.

Reply:
Let's open a separate PR with just this change.

Reply:
Done in https://github.com/graphql-java-kickstart/graphql-java-tools/pull/472